src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -79,7 +79,7 @@ public void setup() throws Exception {
     @Test
     public void indexSnapshotTest() throws Exception {
         // Mock snapshot table data
-        when(bigQueryPdao.getSnapshotTableData(snapshot, searchIndexRequest.getSql()))
+        when(bigQueryPdao.getSnapshotTableData(any(Snapshot.class), any(String.class)))
```

Comment:
Due to the change above I'm not sure how best mock these objects.

Reply:
If we have a separate test that verifies the `translateToTim()` method, then I think it's OK to ignore the query here. Or, you could make `translateToTim()` visible for testing and call it from here.

Reply:
I'm not sure about making it public since it doesn't seem like it would be broadly useful. Maybe I should add the `translateToTim` (now `encodeTimNames`) as part of the `TimUtils` instead? And then it could take a `Map` as a second parameter.

Reply:
I decided to do this, I think it makes more sense.